package/cpp/api/JsiSkPath.h

```diff
@@ -480,8 +480,8 @@ class JsiSkPath : public JsiSkWrappingSharedPtrHostObject<SkPath> {
     auto cmds = jsi::Array(runtime, path.countVerbs());
     auto it = SkPath::Iter(path, false);
     //                       { "Move", "Line", "Quad", "Conic", "Cubic", "Close", "Done" };
-    const int pointCount[] = {     1 ,     2 ,     3 ,      3 ,      4 ,      1 ,     0  };
-    const int cmdCount[] =   {     3 ,     5 ,     7 ,      8 ,      9 ,      3 ,     0  };
+    const int pointCount[] = {     1 ,     1 ,     2 ,      2 ,      3 ,      0 ,     0  };
```

Comment:
The iterator was returning the last point from the previous command as the first point. We corrected that. We might still provide our own JavaScript operator to have that format for convenience.